backend/README.md

```diff
@@ -26,11 +26,17 @@ If you use the config from this repository:
 * Swagger docs will be available at `http://localhost:9000/docs/`.
 
 Note that you need to also run PostgreSQL server to make backend work.
-One way to run it is to use `docker-compose.yml` provided in the [deployment](../deployment) folder.
+One way to run it is to use `docker-compose.yml` provided in the [deployment](../deployment) folder (do not forget to set environment variables related to postgres - create `.env` file from example file [.env.example](../deployment/.env.example))
```

Comment:
Perhaps it can be reverted now, because creating `.env` files is no longer necessary after recent changes.